app/main.py

```diff
@@ -1,8 +1,41 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: List[Car]) -> float:
```

Comment:
I don't know what version of Python you use but for me it works fine without ```from typing import List```, just ```cars: list[Car]```. My version of Python is 3.11, so if my approach doesn't work for you, don't need to fix it.

Reply:
py 3.10: F821 undefined name 'List'